neurovault/apps/statmaps/views.py

```diff
@@ -350,6 +352,11 @@ def upload_folder(request, collection_cid):
                             request.FILES['file'] = f
                             populate_nidm_results(request,collection)
                             continue
+                        elif fnmatch(f.name,'*.feat.zip'):
```

Comment:
This is not how I imaging people will use it. FEAT directories will be given to us either through "Upload directory" mechanism or "Upload archive" (where we should not expect special .feat.zip extension). Thus we need to try to detect if the uploaded folder structure is FEAT or not.